### ISSUE TITLE
data split

### ISSUE THREAD

**Issue by sydat2701** [zhuye98/ICL#8]
Hi,
Could you help me about this problem? As your paper mentioned that "In this work, we only use 300 CT scans from the AMOS 2022 official grand challenge (Ji et al., 2022), including 200 scans for training and 100 for validation. In detail, we divide the original validation set in a ratio of 3:7 to obtain the validation set and test set. ". However, in the split you provided at "data/AMOS22", there are 50 cases for validation, this means that you also have 50 cases for testing, which is different from the mention above. Can you provide the accurate split? 

Thank you

**Comment by zhuye98** [comment]
Hi, 
Thank you for your attention~

We divided 100 scans (from original validation set for segmentation task) in a ratio of 3:7 for validation (30 scans) and test (70 scans). The split file we used is from 'data/AMOS22/dataset_semi_ct.json'. 

So if you check the code in 'train_inherent_consistent_unet_3D_AMOS22.py', you will notice that the --val_num is set to 30. During training, the val_files is [:val_num] and during testing is [val_num:]. Thanks~

**Comment by sydat2701** [comment]
Hi,
so for 100 validation scans, you have the first 30 scans for validation and the remaining 70 for testing? Or they are randomly split in that ratio?

**Comment by zhuye98** [comment]
"you have the first 30 scans for validation and the remaining 70 for testing?"
Yes.
" they are randomly split in that ratio?"
Already randomly split and saved in the 'data/AMOS22/dataset_semi_ct.json'.

**Comment by sydat2701** [comment]
Hi, tks for help! After training your method on AMOS22 dataset with the same split as you describe above, I only got a mean dice score of 54.09% on test set. What is the problem?

**Comment by zhuye98** [comment]
Hi,
Theoretically, if all your experimental settings, including training environment and GPU, are the same as in the paper, you should get the same experimental results (the average dice score of 61.44 for 15 labeled cases in the AMOS dataset in the paper). In your case, I am not sure what the problem is...

**Comment by sydat2701** [comment]
Thank you for your time, and the last question is If I could cite your paper using the same split on AMOS22, which one should I refer to? Cause I saw "15 scans" and "30 scans" in the result table.

**Comment by zhuye98** [comment]
It depends on which semi-supervised setting you use for training. For example, **15 scans** means using 15 scans as labeled data and the **remaining 185 scans** as unlabeled data.
